config/sync/search_api.index.default_solr_index.yml

```diff
@@ -356,8 +357,9 @@ field_settings:
           page: ''
           webform: ''
         strawberryfield_flavor_datasource:
-          digital_object: search_index
-          digital_object_collection: search_index
+          digital_object: ''
```

Comment:
@aksm cool. So... why still digital_object and the other bundles, are returning those values?

Reply:
@DiegoPino Thinking about it now a little more carefully I'm not entirely sure. I was just copying the empty values from rendered_item above. Should they just be removed entirely?